src/components/Events/EventCard.jsx

```diff
@@ -62,37 +65,37 @@ const EventCard = ({
     { timeStyle: 'short' },
   )}`;
 
+  let ref = useRef();
+
+  const breakpoint = 400;
+  const [containerWidth, setContainerWidth] = useState(0);
+
+  useEffect(() => {
+    const observer = new ResizeObserver(entries => {
+      setContainerWidth(entries[0].contentRect.width);
+    });
+    observer.observe(ref.current);
+    return () => ref.current && observer.unobserve(ref.current);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The ref value 'ref.current' will likely have changed by the time this effect cleanup function runs. If this ref points to a node rendered by React, copy 'ref.current' to a variable inside the effect, and use that variable in the cleanup function.